### ISSUE TITLE
A11y_AzureDataStudio_Notebooks-Create_ScreenReader: Not able to perform any action on the toolbar when ORCA screen reader is turned on

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22672]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG1.3.1; #Linux; #Element:EditField; #ScreenReader; #ORCA;

### Actual Result:​
Not able to perform any action on the toolbar when ORCA screen reader is turned on.

### Expected Result:​
User should be able to perform action on the toolbar when ORCA screen reader is turned on.

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "File" and hit ENTER key. TAB to "New Notebook" and hit ENTER key.
3. TAB to "Cell" dropdown and hit ENTER. TAB to "Cell Code" / "Cell Text" and hit ENTER key.
4. Start ORCA screen reader.
5. Navigate to the toolbar using keyboard keys and observe that Not able to perform any action on the toolbar when ORCA screen reader is turned on.

### User Impact:
Screen reader users will not be able to perform any action using toolbar and will not be able to do further task.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:
[228844351-b1650234-bcd3-4323-b1ce-b26f0343793a.webm](https://user-images.githubusercontent.com/90901519/231121948-0dc238b0-e0a8-4c5a-85ae-3d1c763ce6a2.webm)

**Comment by PoojaNamde** [comment]
Primary bug: https://github.com/microsoft/azuredatastudio-mysql/issues/181

**Comment by chlafreniere** [comment]
@PoojaNamde I'm not familiar with Orca. I know on Mac, when Voiceover is turned on, Enter does not work (but this is by design). I believe I need to press CTRL+Option+Space.

Does a similar keybinding exist for Orca while in toolbars?

**Comment by Swati1700** [comment]
@chlafreniere chlafreniere We have checked with CTRL+Win+Space, CTRL+Shift+Space for Orca but still this short cut keys are not working.

**Comment by alanrenmsft** [comment]
@chlafreniere finally I got a chance to look into this issue and Linux with Orca, and it turns out not related to Linux or Orca, I can reproduce the issue on Windows as well.

1. Add a Cell
2. Turn on `Tab Key Moves focus`
3. Move the focus to the editor and press SHIFT+TAB until it reaches the delete button
4. press Space, and nothing happens.

also the toolbar is behaving weird when navigating using keyboard, when the focus is on Delete button, if you navigate with left/right arrow keys, it will never reach the `+` button and the `...` button

could you please take a look at it? BTW, we have a month left until this becomes out of SLA.

**Comment by Charles-Gagnon** [comment]
@PoojaNamde The fix has just been checked in - starting tomorrow you can use the latest insiders build to verify that the issue is resolved.
